doc/c64.sgml

```diff
@@ -116,6 +116,40 @@ cl65 -o file.prg -u __EXEHDR__ -t c64 -C c64-asm.cfg source.s
 Please note that in this case a changed start address doesn't make sense,
 since the program must be loaded to the BASIC start address.
 
+<sect>Extras<p>
+
+<sect1>80 Columns conio driver<p>
+
+The C64 package comes with an alternative software driven 80 columns
+module <tt/c64-soft80.o/ which uses the memory under I/O between &dollar;d000
+and &dollar;ffff.
+
+In memory constrained situations the memory from &dollar;400 to &dollar;7FF
+can be made available to a program by calling <tt/_heapadd ((void *) 0x400, 0x400);/
+at the beginning of <tt/main()/. Doing so is beneficial even if the program
+doesn't use the the heap explicitly because loading a driver (and in fact
+already opening a driver file) uses the heap implicitly.
```

Comment:
The `(and in fact already opening a driver file)`is specific to the Apple II file I/O and not true here.